services/QuillLMS/engines/evidence/app/models/evidence/research/gen_ai/llm.rb

```diff
@@ -78,6 +79,8 @@ def self.g_eval = find_by(vendor: GOOGLE, version: Evidence::Gemini::Chat::SMALL
         def self.form_options = production.where(reasoning: false) + non_production.active.where(reasoning: false).ordered
         def self.all_ordered = production.where(reasoning: false) + non_production.where(reasoning: false).ordered
 
+        def self.in_use = joins(trials: { llm_prompt: :activated_llm_prompts }).distinct
```

Comment:
I think this needs a scope for `active` activated_llm_prompts

Reply:
Good point.  Definitely don't want noise on those.